qqq-backend-core/src/main/java/com/kingsrook/qqq/backend/core/processes/implementations/tablesync/AbstractTableSyncTransformStep.java

```diff
@@ -72,33 +76,33 @@
  *******************************************************************************/
 public abstract class AbstractTableSyncTransformStep extends AbstractTransformStep
 {
-   private static final QLogger LOG = QLogger.getLogger(AbstractTableSyncTransformStep.class);
+   protected static final QLogger LOG = QLogger.getLogger(AbstractTableSyncTransformStep.class);
```

Comment:
i thiiink, in general, we wouldn't want a protected LOG object, but, rather, subclasses to define their own - so that log messages would come out with the subclass's name, not the base class's?  I think?